packages/usa-table/src/usa-table.stories.js

```diff
@@ -20,6 +20,11 @@ export default {
       control: { type: "boolean" },
       defaultValue: false,
     },
+    table_footer: {
+      name: "Include table footer",
+      control: { type: "boolean" },
+      defaultValue: false,
+    }
```

Comment:
Added a control so that we can see the table footer if we want, but it is turned off by default